etc/config/cuda.amazon.properties

```diff
@@ -120,6 +120,54 @@ compiler.hiptrunk.semver=trunk
 compiler.hiptrunk.exe=/opt/compiler-explorer/clang-trunk/bin/clang++
 compiler.hiptrunk.objdumper=/opt/compiler-explorer/clang-trunk/bin/llvm-objdump
 
+group.nvrtc.compilers=nvrtc115:nvrtc114u1:nvrtc114:nvrtc113u1:nvrtc113:nvrtc112u2:nvrtc112u1:nvrtc112:nvrtc111u1:nvrtc111:nvrtc11u1:nvrtc11:nvrtc102:nvrtc101u2:nvrtc101u1:nvrtc101:nvrtc100:nvrtc92:nvrtc91
+group.nvrtc.compilerType=nvcc # nvrtc_cli is sufficiently compatible with nvcc
+group.nvrtc.isSemVer=true
+group.nvrtc.baseName=NVRTC
+group.nvrtc.includeFlag=-I
+group.nvrtc.objdumper=/opt/compiler-explorer/cuda/11.5.0/bin/nvdisasm
+group.nvrtc.rpathFlag=-L # WAR, really need `-Xcompiler "-Wl,-rpath=<path>"`, but not required because supportsExecute=false
```

Comment:
Have you been able to test this? As far as I recall my `config` parser doesn't support `#` comments on the same line as the data, and I worry that this would pass `-L # WAR, ...` etc to the compiler

Reply:
Your parser appears to support it: https://github.com/compiler-explorer/compiler-explorer/blob/f2c1e0bd315fbce55582c440f2d3c8c922a5083a/lib/properties.js#L66
I've tested it and haven't had any problems. The existing nvcc config has had the same line for quite some time too (admittedly that was also added by me).

Reply:
Oh my! I had completely forgotten! Wow; thank you so much for taking the time to check 😊 -- apparently that line dates back to the initial import fo the project ten years ago...so... it has always supported this 😀